docs/runtime_attributes.md

```diff
@@ -0,0 +1,19 @@
+# Key Runtime Attributes
+
+This doc is intended to capture the key runtime attributes for Node.js.  
+These attributes will be used along with the use-cases in order to
+guide what we develop/run as part of the community benchmarking
+efforts.
+
+# Attributes
+
+The following attributes have been identified so far:
+
+* startup/stop time
+* memmory footprint at startup
```

Comment:
allocation througput is mostly interesting if you include the actual application in the startup (assuming e.g. it loads data during startup, or creates some large data structures).